dspace-api/src/main/java/org/dspace/external/service/ExternalDataService.java

```diff
@@ -75,6 +75,7 @@
      * @throws AuthorizeException   If something goes wrong
      * @throws SQLException         If something goes wrong
      */
-    Item createItemFromExternalDataObject(Context context, ExternalDataObject externalDataObject, Collection collection)
+    WorkspaceItem createWorkspaceItemFromExternalDataObject(Context context, ExternalDataObject externalDataObject,
```

Comment:
The JavaDocs of this method need to be updated.  They currently say:
> This method will create an Item in the given Collection based on the given ExternalDataObject.
> Note that this Item will be Archived

You've modified it to return a WorkspaceItem which is *not* Archived.  I see in the below code you've moved where the Item is archived...that's fine, but we should correct the JavaDocs

Reply:
This has been updated